src/analyses/baseInvariant.ml

```diff
@@ -140,7 +139,7 @@ struct
               match ID.to_int n with
               | Some n ->
                 (* When x != n, we can return a singleton exclusion set *)
-                if M.tracing then M.tracec "invariant" "Yes, %a is not %s\n" d_lval x (BI.to_string n);
+                if M.tracing then M.tracec "invariant" "Yes, %a is not %s\n" d_lval x (Z.to_string n);
```

Comment:
We should use the opportunity to add `GobZ.pretty` and use it here (and some other places). It would avoid unnecessary string conversions when output is not needed.